package.json

```diff
@@ -12,8 +12,9 @@
     "electron-menubar": "^1.0.1",
     "electron-squirrel-startup": "^1.0.0",
     "file-extension": "^4.0.0",
+    "go-ipfs-dep": "^0.4.13",
     "ipfs-geoip": "^2.3.0",
-    "ipfsd-ctl": "^0.26.0",
+    "ipfsd-ctl": "ipfs/js-ipfsd-ctl#feat/remote-daemon",
```

Comment:
Should point this to a commit directly instead of a branch. Branches sometimes gets deleted when PRs are merged, or mutates in general. Pointing to the commit prevents those things from having a bad impact.

Reply:
@VictorBjelkholm I'll update this to the right version as soon as it is released and before merging 😄